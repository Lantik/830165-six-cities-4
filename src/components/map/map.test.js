import React from 'react';
import renderer from 'react-test-renderer';
import Map from './map.jsx';

const mock = {
  coords: [
    [52.3909553943508, 4.85309666406198],
    [52.369553943508, 4.85309666406198],
    [52.3909553943508, 4.929309666406198],
    [52.3809553943508, 4.939309666406198]
  ],
  className: `test__map`
};

it(`<Map/> rendered correctly`, () => {
  const {coords, className} = mock;
  const mockedHtmlElement = document.createElement(`div`);

  const tree = renderer
    .create(<Map
      coords={coords}
      className={className}
    />, {
      createNodeMock: () => {
        return mockedHtmlElement;
      }
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
