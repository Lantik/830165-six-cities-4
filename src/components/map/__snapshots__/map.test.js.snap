// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Map/> rendered correctly 1`] = `
<section
  className="cities__map map"
  id="map"
/>
`;
